src/custom/pages/Profile/index.tsx

```diff
@@ -204,6 +207,15 @@ export default function Profile() {
                 )}
               </ButtonPrimary>
             </ConvertWrapper>
+
+            <CardActions>
+              <ExtLink href={getBlockExplorerUrl(chainId, V_COW_CONTRACT_ADDRESS[chainId], 'address')}>
+                Contract ↗
+              </ExtLink>
+              <CopyHelper toCopy={V_COW_CONTRACT_ADDRESS[chainId]}>
+                <div title="Click to copy token contract address">Copy contract</div>
```

Comment:
<img width="1073" alt="image" src="https://user-images.githubusercontent.com/2352112/162184477-122e1eeb-dc65-496a-83d6-52b4ff13f685.png">


Are we sure we prefer to copy the contract address over sending the user to Etherscan?

Reply:
In this case it was not added as a preference per se: You've got both options there (View Contract and Copy Contract).

Rather it was a ported action from the COW balance card, where if you are not connected with MetaMask we show you 'Copy Contract' instead of 'Add Token' as a fall back.

Discussed with @elena-zh that it might not make sense to have 'Add Token' (to MetaMask) for vCOW (given it's non transferable anyway..) but might be OK to have the 'Copy contract' action.

Let me know your thoughts.